.github/workflows/ci.yml

```diff
@@ -47,8 +47,8 @@ jobs:
       postgres:
         image: postgres:latest
         env:
-          POSTGRES_PASSWORD: password
-          POSTGRES_USER: root
+          POSTGRES_PASSWORD: 'postgres'
+          POSTGRES_USER: 'postgres'
```

Comment:
Are these changes still needed?

Reply:
https://github.com/archtechx/tenancy/actions/runs/5044637675/jobs/9047926168#step:2:357

```
Error: Database is uninitialized and superuser password is not specified.
          You must specify POSTGRES_PASSWORD to a non-empty value for the
          superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
```

Looks like we need at least the password

Reply:
We shouldn't need these changes, but it seems that in the tests, we're attempting to log in as the "postgres" user regardless of the user specified here. I'm trying to figure out why right now

Reply:
So the problem was [here](https://github.com/archtechx/tenancy/pull/1105/files#diff-abf1fa77a19f720052b75bc802df0260157b9064d8c3c12a80794f01e9acc065R94-R95) – I hardcoded the credentials. I updated the credentials, things should work OK without this `ci.yml` change. I'm not sure if this env variable setup is correct in the end (e.g. using POSTGRES_USER x DB_USERNAME), I'll take another look at that.

Reply:
>  I'm not sure if this env variable setup is correct in the end (e.g. using POSTGRES_USER x DB_USERNAME)

I decided not to use `DB_USERNAME` as the default for `database.connections.pgsql.username` – I just used the default credentials from `docker-compose.yml`. This will work both with CI and with user's custom credentials set in .env (`POSTGRES_USER`, `POSTGRES_PASSWORD`)

Reply:
So in the end, these changes aren't needed – I'm resolving this

Reply:
When you leave a lot of comments on a review you can keep it unresolved so that I see the thread and can read it.

The file is still changed, so now the question is if the new changes are needed/what are they needed for.

Reply:
> The file is still changed

It isn't, I reverted the changes

Reply:
<img width="1690" alt="CleanShot 2023-05-29 at 8 33 51@2x" src="https://github.com/archtechx/tenancy/assets/33033094/86e49540-d9c9-435e-a891-2bfd5772ed0e">

I was referring to the files tab.

Reply:
For the record, I wrote about the `docker-compose.yml` file on Discord

Reply:
All `ci.yml` and `docker-compose.yml` changes are reverted now